lib/initial-setup/WP_Auth0_InitialSetup_Connections.php

```diff
@@ -3,239 +3,18 @@
 class WP_Auth0_InitialSetup_Connections {
 
 	protected $a0_options;
-	protected $providers = array(
```

Comment:
We no longer control connections from the settings page so this is not needed. 

The `options` child array is still used in a method for this class - `WP_Auth0_InitialSetup_Connections::toggle_social()` - so those 3 were moved down. Method will be deprecated in another PR but need to retain functionality in the meantime.